pkg/kotsadm/types/constants_test.go

```diff
@@ -123,3 +123,33 @@ func Test_mergeLabels(t *testing.T) {
 		})
 	}
 }
+
+func Test_labelsToStrings(t *testing.T) {
+	tests := []struct {
+		name   string
+		labels map[string]string
+		expect string
+	}{
+		{
+			name: "pass case with labels",
+			labels: map[string]string{
+				"foo": "foo",
+			},
+			expect: "foo=foo",
+		},
+		{
+			name: "pass case with troubleshoot labels",
+			labels: map[string]string{
+				"troubleshoot.io/kind": "support-bundle",
+			},
+			expect: "troubleshoot.io/kind=support-bundle",
+		},
```

Comment:
maybe add another case for testing multiple labels render as expected?

Reply:
more tests has been added for testing multiple labels